remote_test.go

```diff
@@ -1220,17 +1221,20 @@ func (s *RemoteSuite) TestGetHaves(c *C) {
 	sto := filesystem.NewStorage(f.DotGit(), cache.NewObjectLRUDefault())
 
 	var localRefs = []*plumbing.Reference{
+		// Exists
 		plumbing.NewReferenceFromStrings(
 			"foo",
-			"f7b877701fbf855b44c0a9e86f3fdce2c298b07f",
+			"b029517f6300c2da0f4b651b8642506cd6aaf45d",
 		),
+		// Exists
 		plumbing.NewReferenceFromStrings(
 			"bar",
-			"fe6cb94756faa81e5ed9240f9191b833db5f40ae",
+			"b8e471f58bcbca63b07bda20e428190409c2db47",
 		),
+		// Doesn't Exist
 		plumbing.NewReferenceFromStrings(
 			"qux",
-			"f7b877701fbf855b44c0a9e86f3fdce2c298b07f",
+			"0000000",
```

Comment:
Having to change this, feels like we are breaking existing/expected behaviour.

Reply:
I think the test was wrong, it was passing in a list of references that didn't actually exist

Reply:
You are right, the previous commits did not exist at [the target](https://github.com/git-fixtures/basic/commits/master/). Thank you for updating and adding comments to that effect.